.buildkite/e2e/nightly-main-matrix.yaml

```diff
@@ -51,11 +52,10 @@
   fixed:
     E2E_PROVIDER: ocp
   mixed:
-    #- DEPLOYER_CLIENT_VERSION: "4.8.57"
-    #- DEPLOYER_CLIENT_VERSION: "4.9.58"
     #- DEPLOYER_CLIENT_VERSION: "4.10.55"
     #- DEPLOYER_CLIENT_VERSION: "4.11.33"
-    - DEPLOYER_CLIENT_VERSION: "4.13.4"
+    #- DEPLOYER_CLIENT_VERSION: "4.13.4"
+    - DEPLOYER_CLIENT_VERSION: "4.14.7"
```

Comment:
Last stable version advised for https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable-4.14/ is `4.14.6`, published on `Tue Dec 19 12:53:46 2023` . There is a version `4.14.7` but it seems to have been released 4 days before: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.14.7/ 🤷

Reply:
Well, I just went here and looked at the most recent by version:
https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/
But I do see your point @barkbay . I tested [yesterday](https://github.com/elastic/cloud-on-k8s/pull/7387#issuecomment-1860736813) with `4.14.7` successfully. Any preference on which version to move forward with? I have no strong opinion either way.

Reply:
I download the last client from RedHat:

Reply:
I downloaded the most recent version from the RedHat web site:

```
oc version
Client Version: 4.14.6
Kustomize Version: v5.0.1
```

I would go with `4.14.6` something looks wrong with `4.14.7`

Reply:
Sounds good. Testing now.